### DISCUSSION TITLE
[Wiki] Documenation and restructuring

### DISCUSSION THREAD

**Discussion by ashthespy** [librespot-org/librespot#863] [General]
We have a lot of old info still scattered around [gitter discussions](https://gitter.im/librespot-org/spotify-connect-resources), the old [`spotify-connect-resources`](https://github.com/librespot-org/spotify-connect-resources) and [`spotify-analyze`](https://github.com/librespot-org/spotify-analyze) repos, not to mention other issues/discussions/PRs over here and [`librespot-java`](https://gitter.im/librespot-org/librespot-java) for the newer API. 

I know documentation is a hot topic, but it would be good to slowly collect and add it somewhere. While the RE/API stuff is up for discussion,  I'd be more interested in the implementation/architecture considerations..

Also would like to propose a wiki tune up - splitting out the different backends into their own pages with their howtos/sane defaults etc..

What do you all think?

**Comment by roderickvd** [comment]
Great idea. Happy to see you bringing it up.

While at it I would also be in favour of doing some work on the GitHub environment. Like setting up issue templates, code of conduct, and doing some milestone planning.

Of course I'd be happy to help.

**Comment by ashthespy** [comment]
Go for it! 👍 Could I also request some help with the wiki? I'd like to simplify the options table a bit, and make it less scary :-D  

W.t.r milestone planning however its best to be pragmatic - this is a FOOS project, and contributors may or may not always have the time to dedicate to it..

**Reply by roderickvd** [reply]
> Go for it! 👍 Could I also request some help with the wiki? I'd like to simplify the options table a bit, and make it less scary :-D

Sure if you start mapping out ideas, just hand me a task.

> W.t.r milestone planning however its best to be pragmatic - this is a FOOS project, and contributors may or may not always have the time to dedicate to it..

Absolutely. I wasn't thinking about setting dates or anything like that. But with `new-api` and some issues that'd require a bit more work, I would find it helpful to keep track of ideas and draft which is targeted for maintenance 0.3.x, upcoming 0.4, and so on.
